Gemfile.lock

```diff
@@ -310,7 +310,7 @@ GEM
       listen (~> 2.7, >= 2.7.3)
     retriable (1.4.1)
     retryable (1.3.5)
-    rmagick (2.13.2)
+    rmagick (2.13.4)
```

Comment:
Right now, `rmagick` actually has no version range specified in the Gemfile, which means that we're not specifying the version we depend on for this gem.

Though `Gemfile` takes semantic-version ranges, Bundler will only install the specific version specified in `Gemfile.lock`, which we control through GitHub to ensure that all of our installed dependencies match up across our environments. Not only is this necessary for our current setup (because we run `bundle install` on our production frontends, rather than a pre-vendored gem bundle), but I do think it helps to be precise about versions we've actually installed on our machines, just to eliminate any question of version inconsistencies.

Having the semantic-version range is still helpful when we want to update things in a controlled way - e.g. we could run `bundle update`, run tests and make sure everything's still working, then commit the updated `Gemfile.lock` to the project.